spec/factories/media.rb

```diff
@@ -72,6 +72,13 @@
       sort { "Kaviar" }
       teachable_sort { :valid_lesson }
       after(:build) { |m| m.editors << m.teachable.lecture.teacher }
+
+      trait :with_lesson_by_id do
+        transient do
+          lesson_id { nil }
+        end
+        teachable { Lesson.find(lesson_id) }
+      end
```

Comment:
Same as above: I would expect an exception for the default value. Is this intended?

Reply:
See above